src/Microsoft.ML.Legacy/CSharpApi.cs

```diff
@@ -3309,6 +3315,11 @@ public sealed partial class OnnxConverter
             /// </summary>
             public Var<Microsoft.ML.Runtime.EntryPoints.ITransformModel> Model { get; set; } = new Var<Microsoft.ML.Runtime.EntryPoints.ITransformModel>();
 
+            /// <summary>
+            /// The targeted ONNX version. It can be either "Stable" or "Experimental"
```

Comment:
What does Stable and Experimental refer to? Should this be documented here?

Reply:
Will do. Thanks.